Formula/vcluster.rb

```diff
@@ -0,0 +1,35 @@
+class Vcluster < Formula
+  desc "Creates fully functional virtual k8s cluster inside host k8s cluster's namespace"
+  homepage "https://www.vcluster.com"
+  url "https://github.com/loft-sh/vcluster.git",
+    tag:      "v0.7.1",
+    revision: "dc0ff6f96e9c96fe2caa77e79c2dffc921b4fd49"
+  license "Apache-2.0"
+  head "https://github.com/loft-sh/vcluster.git", branch: "main"
+
+  depends_on "go" => :build
+  depends_on "helm"
+  depends_on "kubernetes-cli"
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X main.commitHash=#{Utils.git_head}
+      -X main.buildDate=#{time.iso8601}
+      -X main.version=#{version}
+    ]
+    system "go", "build", "-mod", "vendor", *std_go_args(ldflags: ldflags), "./cmd/vclusterctl/main.go"
+  end
```

Comment:
I think you can also do the completion setup

```
(zsh_completion/"_vcluster").write Utils.safe_popen_read(bin/"vcluster", "completion", "zsh")
(bash_completion/"vcluster").write Utils.safe_popen_read(bin/"vcluster", "completion", "bash")
```

https://github.com/loft-sh/vcluster/blob/main/cmd/vclusterctl/cmd/completion.go

Reply:
Thanks, added.